openquake/calculators/tests/outputs/realizations.csv

```diff
@@ -1,2 +1,3 @@
 rlz_id,branch_path,weight
-0,"[FromShakeMap]",1.0000000e+00
+0,"[LinLee2008SSlab]\nminimum_distance = 10",6.0000002e-01
```

Comment:
Is the newline character expected here in the GMPE name?

Reply:
Yes, because inside the realization output we do not store only the GMPE name but also the GMPE arguments. Think for instance to the GMPEs for Canada, which depends on an HDF5 file: they are all instances of GMPETable but with different arguments.